### PR TITLE
Update Line to fix move function

### PR DESCRIPTION
**Pull request by MatthewMeacham**
Mimicked what was happening in Triangle in here to fix the move issue so that the origin would always be the top left.

### CODE REVIEW

File: src/app/shapes/Line.ts [hunk 1]
```diff
@@ -6,33 +6,28 @@ export class Line extends Shape {
   private strokeWidth: number = 2;
   private static DEFAULT_COLOR: string = "black";
 
-  private x1: number;
-  private y1: number;
-  private x2: number;
-  private y2: number;
-  
   public constructor(name: string, x1: number, y1: number, x2: number, y2: number) {
```
Comment (MicahElizabeth) [inline src/app/shapes/Line.ts:9]: If you use a Konva.Line you won't need these.

Comment (MicahElizabeth) [reply src/app/shapes/Line.ts:9]: They would get sent in points

Comment (MatthewMeacham) [reply src/app/shapes/Line.ts:9]: This PR removed those..?

**Review by allisoninman — APPROVED** [review]

**Review by MicahElizabeth — APPROVED** [review]

File: src/app/shapes/Line.ts (hunk 1 above)
Comment (MicahElizabeth) [reply src/app/shapes/Line.ts:9]: 🤦‍♀️

**Pull request merged**